client/packages/programs/src/JsonForms/components/ProgramSearch.tsx

```diff
@@ -18,6 +18,7 @@ const UIComponent = (props: ControlProps) => {
   const onChange = async (program: ProgramFragment) => {
     setProgram(program);
     handleChange(path, program.id);
+    handleChange('elmisCode', program.elmisCode);
```

Comment:
changes for elmis code was needed for https://github.com/msupply-foundation/open-msupply-reports/issues/150... sorry have snuck into this small PR